cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1454,6 +1453,11 @@ mod benches {
 	);
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+pub const RANDOM_PARA_ID: ParaId = ParaId::new(43211234);
```

Comment:
please, use the same pattern everywhere: `pub const RandomParaId: ParaId = ParaId::new(43211234);`


```suggestion
```